airbyte-api/src/main/openapi/config.yaml

```diff
@@ -3744,7 +3744,24 @@ components:
           description: Alias name to the stream to be used in the destination
           type: string
         selected:
+          description: If this is true, the stream is selected with all of its properties.
           type: boolean
+        fieldSelectionEnabled:
+          description: Whether field selection should be enabled. If this is true, only the properties in `selectedFields` will be included.
```

Comment:
should we say 'included in the configured catalog'? maybe that's already clear, though - just a suggestion

Reply:
Done.